README.md

```diff
@@ -51,19 +54,38 @@ everything, you can use the second parameter of the `copy` method:
 // Pass false as second parameter to force copying everything rather than doing an incremental copy.
 $databaseCopier->copy($contactZohoDao, false);
 ```
+-----------------
+<h4>ZohoDatabaseSyncZoho</h4>
+
+Usage:
 
+```php
+// $connection is a Doctrine DBAL connection to your database.
+$databaseSync = new ZohoDatabaseSyncZoho($connection);
+
+// $contactZohoDao is the Zoho Dao to the module you want to copy.
+//pushInsertedRows method to push the new rows to Zoho
+$databaseSync->pushInsertedRows($contactZohoDao);
+//pushInsertedRows method to push the updated rows to Zoho
+$databaseSync->pushUpdatedRows($contactZohoDao);
+//pushDeletedRows method to delete the deleted rows to Zoho
+$databaseSync->pushDeletedRows($contactZohoDao);
+```
```

Comment:
We should make a pass to rename objects / methods.

Maybe something like:

`ZohoDatabaseCopier::copy` => `ZohoDatabaseCopier::fetchFromZoho`

`ZohoDatabaseSyncZoho::xxx` => `ZohoDatabasePusher::pushToZoho` (the `pushToZoho` method calls in turn the pushInertedRows, pushUpdatedRows and pushDeletedRows). This makes a cleaner public interface to the class.